core/src/main/scala/io/odin/formatter/Formatter.scala

```diff
@@ -70,49 +73,45 @@ object Formatter {
       val iterator = context.iterator
       while (iterator.hasNext) {
         val (key, value) = iterator.next()
-        builder.append(p"$key: $value")
+        builder.append(s"$key: $value")
         if (iterator.hasNext) builder.append(", ")
       }
       builder.toString()
     }
 
-  /**
-    * Formats timestamp using the following format: yyyy-MM-ddTHH:mm:ss,SSS
+  /** Formats timestamp using the following format: yyyy-MM-ddTHH:mm:ss,SSS
     */
   def formatTimestamp(timestamp: Long): String = {
     val date = timestamp.t
-    p"${date.F}T${date.T},${date.milliOfSecond}"
+    s"${date.F}T${date.T},${date.milliOfSecond}"
   }
 
-  /**
-    * The result differs depending on the format:
+  /** The result differs depending on the format:
     *
-    * `PositionFormat.Full` - prints full position
-    * 'io.odin.formatter.Formatter formatPosition:75' formatted as 'io.odin.formatter.Formatter formatPosition:75'
+    * `PositionFormat.Full` - prints full position 'io.odin.formatter.Formatter formatPosition:75' formatted as
+    * 'io.odin.formatter.Formatter formatPosition:75'
     *
-    * `PositionFormat.AbbreviatePackage` - prints abbreviated package and full enclosing
-    * 'io.odin.formatter.Formatter formatPosition:75' formatted as 'i.o.f.Formatter formatPosition:75'
+    * `PositionFormat.AbbreviatePackage` - prints abbreviated package and full enclosing 'io.odin.formatter.Formatter
+    * formatPosition:75' formatted as 'i.o.f.Formatter formatPosition:75'
     */
   def formatPosition(position: Position, format: PositionFormat): String = {
-    val lineNumber = if (position.line >= 0) p":${position.line}" else ""
+    val lineNumber = if (position.line >= 0) s":${position.line}" else ""
 
     val enclosure = format match {
       case PositionFormat.Full              => position.enclosureName
       case PositionFormat.AbbreviatePackage => abbreviate(position.enclosureName)
 
     }
 
-    p"$enclosure$lineNumber"
+    s"$enclosure$lineNumber"
   }
 
-  /**
-    * Default Throwable printer is twice as slow. This method was borrowed from scribe library.
+  /** Default Throwable printer is twice as slow. This method was borrowed from scribe library.
     *
-    * The result differs depending on the format:
-    * `ThrowableFormat.Depth.Full` - prints all elements of a stack trace
-    * `ThrowableFormat.Depth.Fixed` - prints N elements of a stack trace
-    * `ThrowableFormat.Indent.NoIndent` - prints a stack trace without indentation
-    * `ThrowableFormat.Indent.Fixed` - prints a stack trace prepending every line with N spaces
+    * The result differs depending on the format: `ThrowableFormat.Depth.Full` - prints all elements of a stack trace
```

Comment:
something is awfully off with formatting of all the docs

Reply:
@sergeykolbasov I fixed the formatting of the docs, but by updating `scalafmt` (skipping many versions to the most recent 3.0.0-RC4 as it has best Scala 3 support), there are some changes that seem unavoidable – they happen on `master` as well. If that's an issue for getting this PR through, we can  update `scalafmt` in a separate PR first (defining the style there) and then the diff of this PR should be more focused on the actual code changes for Scala 3. What do you prefer?

Reply:
Hmm, I don't mind style-driven commits as far as it's tooling-based, but I think `scalafmt` did something wrong in general. I'll check their latest release to see if there is a bug or configuration missing, but breaking docs newlines is definitely something that the formatting tools shouldn't do.

Reply:
ah, you already did

nice